setuptools/tests/test_egg_info.py

```diff
@@ -855,6 +855,33 @@ def test_long_description_content_type(self, tmpdir_cwd, env):
         assert expected_line in pkg_info_lines
         assert 'Metadata-Version: 2.1' in pkg_info_lines
 
+    def test_long_description(self, tmpdir_cwd, env):
+        # Test that specifying `long_description` and `long_description_content_type`
+        # keyword args to the `setup` function results in writing
+        # the description in the message payload of the `PKG-INFO` file
+        # in the `<distribution>.egg-info` directory.
+        self._setup_script_with_requires(
+            "long_description='This is a long description\\nover multiple lines',"
+            "long_description_content_type='text/markdown',"
+        )
+        environ = os.environ.copy().update(
+            HOME=env.paths['home'],
+        )
```

Comment:
`dict.update()` does not return anything, it updates the dict in-place. Meaning that `environ=None` always in this test.

I believe what you meant to do is:
```suggestion
        environ = dict(os.environ, HOME=env.paths['home'])
```

Reply:
Tbh I just copied that part from the test above without taking a closer look 😅 Turns out it works perfectly even completely without it.

Reply:
Interesting. So this means there's a bug in other tests...